File: examples/dotnet/BackgroundRemoval.cs
using AILabTools;
using AILabTools.Generated.Params;
using AILabTools.Http;

using var client = new AILabToolsClient(
    Environment.GetEnvironmentVariable("AILAB_API_KEY")!);

var result = await client.Background.RemoveAsync(
    new CutoutUniversalBackgroundRemovalParams
    {
        Image = FileInput.FromPath("./image.jpg"),
        ReturnForm = "whiteBK",
    });

Console.WriteLine(result.Data?.ImageUrl);

File: examples/ruby/background_removal.rb
# frozen_string_literal: true

require "ailabtools"

client = AILabTools::Client.new(api_key: ENV.fetch("AILAB_API_KEY"))
params = AILabTools::Generated::Params::CutoutUniversalBackgroundRemovalParams.new(
  image: AILabTools::FileInput.from_path("image.jpg", content_type: "image/jpeg"),
  return_form: "whiteBK"
)

result = client.background.remove(params)
puts result.data&.image_url
